docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -1,63 +1,79 @@
 
-== Connector updates
+For more info, including how to use the 'JobBuilder' API, see xref:pipelines:job-placement-control.adoc[].
 
-**Kafka Connect Source connector** is now generally available, which means that Kafka Connect sources can scale across the whole cluster instead of just a single member. For more information, see xref:integrate:kafka-connect-connectors.adoc[].
+== Multi-member routing for Java clients
 
-**Mongo SQL Connector (SQL over MongoDB)** is now generally available. With the addition of security support, you can now secure your data connection with TLS when creating MongoDB mappings over SQL. For more information, see xref:sql:mapping-to-mongo.adoc[].
+Geographically dispersed or 'stretched' clusters are not as resilient as standard HA clusters, and also suffer from lower throughput, longer recovery and longer operations. With the release of Hazelcast Platform 5.5, you can use multi-member routing to provide greater performance and stability for Java client applications connecting to geographically dispersed clusters. This means much faster transaction re-routing in the event of an outage. You can implement strong consistency on a stretch cluster and reduce connection overheads and improve overall performance. You can easily enable client multi-member routing using your existing client network configuration and get these benefits, all with zero downtime.
 
-**JDBC SQL Connector (SQL over JDBC)** is now generally available. In addition to MySQL and PostgreSQL, the JDBC SQL connector now also supports Oracle and Microsoft SQL Server databases. For more information, see xref:sql:mapping-to-jdbc.adoc#data-type-mapping-between-hazelcast-and-mssql[Mapping between Hazelcast and MSSQL].
+Before 5.5, Hazelcast clients could only have one of two types of connection to a cluster: single member (also known as unisocket) or all member. With a unisocket connection the client only connected to a single cluster member, which meant all communication was relayed via the connected cluster member and could therefore suffer from performance issues. With an all member connection the client connects to every member in the cluster and is partition-aware, which means that communications are sent directly to the cluster member holding the required data. If you're using partition groups to isolate data to specific cluster members, there was previously no way for the client to have visibility into this configuration and therefore was not connecting as efficiently as possible, or adapting to failovers. For example, CP Subsystem uses a group leader - clients would not be aware of this and there would an extra internal relay stage rather than sending directly to the group leader alone.
```

Comment:
```suggestion
Before 5.5, Hazelcast clients could only have one of two types of connection to a cluster: single member (also known as unisocket) or all member (also known as smart). With a unisocket connection the client only connected to a single cluster member, which meant all communication was relayed via the connected cluster member and could therefore suffer from performance issues. With an all member connection the client connects to every member in the cluster and is partition-aware, which means that communications are sent directly to the cluster member holding the required data. If you're using partition groups to isolate data to specific cluster members, there was previously no way for the client to have visibility into this configuration and therefore was not connecting as efficiently as possible, or adapting to failovers. For example, CP Subsystem uses a group leader - clients would not be aware of this and there would an extra internal relay stage rather than sending directly to the group leader alone.
```

Reply:
Just to tie the two previous config options; not vital!